tck/src/main/java/org/eclipse/microprofile/config/tck/ArrayConverterTest.java

```diff
@@ -1654,120 +1658,161 @@ public void testInstantSetInjection() {
 
     ///////////////////////////////////Test URL[] //////////////////////////
 
+    private void assertURLArrayEquals(URL[] value) throws MalformedURLException {
+        URL[] expectedValue = new URL[]{
+            new URL("http://microprofile.io"),
+            new URL("http://openliberty.io"),
+            new URL("http://microprofile.io")};
+        Assert.assertTrue(IntStream.range(0, expectedValue.length).allMatch(i -> {
+        try {
+                return expectedValue[i].toURI().equals(value[i].toURI());
+        } 
+        catch (URISyntaxException e) {
+                e.printStackTrace();
+        }
+        return false;
+        }));
+    }
+
+    private void assertURLListEquals(List<URL> value) throws MalformedURLException {
```

Comment:
I'd think that either of the `assertURLArrayEquals` and `assertURLListEquals` could be implemented in terms of the other. I'd do `assertURLArrayEquals` => `assertURLListEquals(asList(value))`